README.md

```diff
@@ -139,9 +139,9 @@ ktmpl ./deploy.yml \
    commits in the merge request as described in the next section.
 
 
-## Adding Reviewed-by: and Tested: messages to commits
+## Adding tags to commits
 Marge-bot supports automated addition of the following
-two [standardized git commit headers](https://www.kernel.org/doc/html/v4.11/process/submitting-patches.html#using-reported-by-tested-by-reviewed-by-suggested-by-and-fixes):
+two [standardized git commit tags](https://www.kernel.org/doc/html/v4.11/process/submitting-patches.html#using-reported-by-tested-by-reviewed-by-suggested-by-and-fixes):
```

Comment:
"standardized commit message trailers"? I wrote "headers" originally (and they look and more or less work like HTTP or SMTP headers), but I think trailers is the way to go.